app/controllers/root_controller.rb

```diff
@@ -50,15 +50,15 @@ def random_page
     # new content.
     expires_in 5.seconds
 
-    base_url = "https://www.gov.uk/api/artefacts.json"
-    total_pages = JSON.parse(RestClient.get(base_url).body)['pages']
+    base_url = "https://www.gov.uk/api/search.json"
+    total_pages = JSON.parse(RestClient.get("#{base_url}?count=0"))['total']
 
     random_page_number = Random.rand(1..total_pages)
```

Comment:
The "start" parameter in rummager is 0-based, so you want a range from (0..total_pages - 1) here.